setup.py

```diff
@@ -13,7 +13,7 @@
 with open(path.join(here, 'README.md'), encoding='utf-8') as f:
     long_description = f.read()
 
-VERSION = "5.4.2"
+VERSION = "5.5"
```

Comment:
Not sure how version numbers are being used in this project, assuming we would want a new version if this were to be merged?